### PR TITLE
Migrate to TypeScript

### PR DESCRIPTION
**Pull request by zainfathoni**

### CODE REVIEW

File: components/sections/pricing.tsx [hunk 1]
```diff
@@ -25,7 +31,11 @@ export const PricingItem = ({ children }) => (
   </li>
 );
 
-export const PricingIncluded = ({ title, children }) => (
+type IncludedProps = {
+  title: string,
+  children: React.ReactNode // TODO: Allow Item type only
```
Comment (zainfathoni) [inline components/sections/pricing.tsx:36]: How can I type this `children` prop to restrict the type of the children passed in?
I want it to only allow children with the `Item` component type only.

File: components/sections/pricing.tsx [hunk 2]
```diff
@@ -39,14 +49,18 @@ export const PricingIncluded = ({ title, children }) => (
   </div>
 );
 
-export const PricingSection = ({ title, children }) => {
+type PricingProps = {
+  title: string,
+  children: React.ReactNode // TODO: Allow Description and Included types only
```
Comment (zainfathoni) [inline components/sections/pricing.tsx:54]: How can I type this `children` prop to restrict the type of the children passed in?
I want it to only allow children with the `Description` and `Included` component types only.

File: components/sections/pricing.tsx (hunk 1 above)
Comment (alam3211) [reply components/sections/pricing.tsx:36]: have you tried this one?
```suggestion
  children: React.ReactElement<ItemProps> | Array<React.ReactElement<ItemProps>>
```

if it's work, it should be works also for below one
```suggestion
  children: React.ReactElement<DescriptionProps> | Array<React.ReactElement<DescriptionProps>>
  React.ReactElement<IncludedProps> | Array<React.ReactElement<IncludedProps>>
```

Comment (zainfathoni) [reply components/sections/pricing.tsx:36]: Thanks for the suggestion, unfortunately, it doesn't prevent me from rendering another type of component, such as `Pricing.Description`. 😅 
Screenshot 2021-04-10 at 9 46 05 PM

**Pull request merged**